.github/workflows/e2e_custom_cl.yml

```diff
@@ -6,7 +6,7 @@ on:
       cl_branch_ref:
         description: Chainlink repo branch to integrate with
         required: true
-        default: develop
+        default: starknet-bump
```

Comment:
actually, we can point this directly at `develop` right now since that's what i've been testing against locally so it should work. the flow will pull the chainlink repo, `go get & go mod tidy` the latest commit on this branch, then build